yaml/ttf07.yaml

```diff
@@ -46,6 +56,9 @@ select:
       then:
         - lambda: |-
             const auto betriebsartId = Mapper::instance().getBetriebsartId(x);
+
+            ESP_LOGI("ttf07.yaml:select", "Updating PROGRAMMSCHALTER to %s betriebsartId=0x%0x", x.c_str(), betriebsartId.value());
```

Comment:
please move that inside 

```
if(betriebsartId.has_value()) {
```
Otherwise your ESP will crash in case the mapping did not work